metrics/web/test/dashboard/presentation/state/project_metrics_notifier_test.dart

```diff
@@ -402,23 +402,19 @@ void main() {
         final projectMetrics = notifier.projectsMetricsTileViewModels.first;
         final buildResultMetrics = projectMetrics.buildResultMetrics;
 
-        final metricPeriodStart = buildResultMetrics.metricPeriodStart;
-        final metricPeriodEnd = buildResultMetrics.metricPeriodEnd;
-
-        expect(metricPeriodStart, isNull);
-        expect(metricPeriodEnd, isNull);
+        expect(buildResultMetrics.dateRangeViewModel, isNull);
       },
     );
 
     test(
-      "loads a build result metric with the metric period start from the latest number of builds to load for chart metrics build results",
+      "loads a build result metric with the date range with the start date from the latest number of builds to load for chart metrics build results",
```

Comment:
```suggestion
      "loads a build result metric with the date range containing the start date from the latest number of builds to load for chart metrics build results",
```
Would it a bit more readable?